src/js/components/tablero/Filter.jsx

```diff
@@ -163,6 +179,13 @@ class Filter extends Component {
                 > X
                 </span>
               </div>
+              <input
```

Comment:
Add value property in this input: `value={this.state.searchTerm}`

Reply:
Done